### ISSUE TITLE
Duplicate IDs should not be flagged as errors when not visible to AT

### ISSUE THREAD

**Issue by thierryk** [dequelabs/axe-core#2910]
> "The value of an ARIA ID must be unique to prevent other instances from being overlooked by assistive technologies."

What if the duplicates are inside a container with `aria-hidden="true"`?

Code example:

```
<div aria-hidden="true">
  <h4 id="foo">Heading Foo</h4>
  <h4 id="bar">Heading Bar</h4>
</div>

<h4 id="foo">Heading Foo</h4>
<h4 id="bar">Heading Bar</h4>
```

I have a use case where tabs in a widget are cloned to help figure out in which state the widget should be ([TabPanel vs. Accordion](https://tabpanelwidget.com)).

It is important that all cloned elements retained their ID to make sure any style associated to these IDs apply to **both instances** (the ones exposed to AT and the ones **hidden** from AT).

I believe this is a valid use case for duplicating IDs hence why I think *axe-core* should not flag those as errors.

**Comment by WilcoFiers** [comment]
Duplicate IDs can cause issues regardless of if they are visible or not. There are quite a few closed issues explaining this. Please see #2808, #2291, or #862.

**Comment by thierryk** [comment]
Thanks for the links to existing issues (I should have done my homework).

> Axe-core is a testing tool for WCAG

Enough said! ;)
